po: Update translations from Transifex

urbalazs opened the pull request:
Fixes https://github.com/p11-glue/p11-kit/issues/378

Updates translations from Transifex using Transifex client and some magic command line commands :)

Before translation update:

```
ar.po: 0 translated messages, 81 untranslated messages.
as.po: 0 translated messages, 81 untranslated messages.
ast.po: 0 translated messages, 81 untranslated messages.
az.po: 0 translated messages, 81 untranslated messages.
bg.po: 0 translated messages, 81 untranslated messages.
bn_IN.po: 0 translated messages, 81 untranslated messages.
ca.po: 81 translated messages.
ca@valencia.po: 0 translated messages, 81 untranslated messages.
cy.po: 0 translated messages, 81 untranslated messages.
cs.po: 81 translated messages.
da.po: 81 translated messages.
de.po: 81 translated messages.
el.po: 81 translated messages.
en_GB.po: 81 translated messages.
eo.po: 14 translated messages, 67 untranslated messages.
es.po: 81 translated messages.
et.po: 0 translated messages, 81 untranslated messages.
eu.po: 0 translated messages, 81 untranslated messages.
fa.po: 0 translated messages, 81 untranslated messages.
fi.po: 81 translated messages.
fo.po: 0 translated messages, 81 untranslated messages.
fr.po: 81 translated messages.
fur.po: 81 translated messages.
ga.po: 0 translated messages, 81 untranslated messages.
gl.po: 81 translated messages.
gu.po: 0 translated messages, 81 untranslated messages.
he.po: 0 translated messages, 81 untranslated messages.
hi.po: 0 translated messages, 81 untranslated messages.
hr.po: 81 translated messages.
hu.po: 81 translated messages.
ia.po: 0 translated messages, 81 untranslated messages.
id.po: 81 translated messages.
it.po: 81 translated messages.
ja.po: 81 translated messages.
ka.po: 31 translated messages, 50 untranslated messages.
kk.po: 4 translated messages, 77 untranslated messages.
kn.po: 0 translated messages, 81 untranslated messages.
ko.po: 81 translated messages.
lt.po: 0 translated messages, 81 untranslated messages.
lv.po: 80 translated messages, 1 untranslated message.
ml.po: 0 translated messages, 81 untranslated messages.
mr.po: 0 translated messages, 81 untranslated messages.
ms.po: 0 translated messages, 81 untranslated messages.
nb.po: 0 translated messages, 81 untranslated messages.
nl.po: 80 translated messages, 1 untranslated message.
nn.po: 0 translated messages, 81 untranslated messages.
oc.po: 81 translated messages.
or.po: 0 translated messages, 81 untranslated messages.
pa.po: 74 translated messages, 7 untranslated messages.
pl.po: 81 translated messages.
pt_BR.po: 81 translated messages.
pt.po: 81 translated messages.
ro.po: 0 translated messages, 81 untranslated messages.
ru.po: 81 translated messages.
sk.po: 81 translated messages.
sl.po: 81 translated messages.
sq.po: 0 translated messages, 81 untranslated messages.
sr@latin.po: 0 translated messages, 81 untranslated messages.
sr.po: 81 translated messages.
sv.po: 81 translated messages.
ta.po: 0 translated messages, 81 untranslated messages.
te.po: 0 translated messages, 81 untranslated messages.
th.po: 0 translated messages, 81 untranslated messages.
tr.po: 81 translated messages.
uk.po: 81 translated messages.
vi.po: 0 translated messages, 81 untranslated messages.
wa.po: 0 translated messages, 81 untranslated messages.
zh_CN.po: 81 translated messages.
zh_HK.po: 0 translated messages, 81 untranslated messages.
zh_TW.po: 1 translated message, 80 untranslated messages.
```

After translation update:

```
ar.po: 0 translated messages, 313 untranslated messages.
as.po: 0 translated messages, 313 untranslated messages.
ast.po: 81 translated messages, 232 untranslated messages.
az.po: 0 translated messages, 313 untranslated messages.
bg.po: 0 translated messages, 313 untranslated messages.
bn_IN.po: 0 translated messages, 313 untranslated messages.
ca.po: 81 translated messages, 232 untranslated messages.
ca@valencia.po: 0 translated messages, 313 untranslated messages.
cy.po: 0 translated messages, 313 untranslated messages.
cs.po: 81 translated messages, 232 untranslated messages.
da.po: 301 translated messages, 8 fuzzy translations, 4 untranslated messages.
de.po: 192 translated messages, 8 fuzzy translations, 113 untranslated messages.
el.po: 81 translated messages, 232 untranslated messages.
en_GB.po: 81 translated messages, 232 untranslated messages.
eo.po: 14 translated messages, 299 untranslated messages.
es.po: 301 translated messages, 8 fuzzy translations, 4 untranslated messages.
et.po: 0 translated messages, 313 untranslated messages.
eu.po: 21 translated messages, 292 untranslated messages.
fa.po: 0 translated messages, 313 untranslated messages.
fi.po: 291 translated messages, 8 fuzzy translations, 14 untranslated messages.
fo.po: 0 translated messages, 313 untranslated messages.
fr.po: 81 translated messages, 232 untranslated messages.
fur.po: 81 translated messages, 232 untranslated messages.
ga.po: 0 translated messages, 313 untranslated messages.
gl.po: 81 translated messages, 232 untranslated messages.
gu.po: 0 translated messages, 313 untranslated messages.
he.po: 0 translated messages, 313 untranslated messages.
hi.po: 0 translated messages, 313 untranslated messages.
hr.po: 127 translated messages, 3 fuzzy translations, 183 untranslated messages.
hu.po: 313 translated messages.
ia.po: 0 translated messages, 313 untranslated messages.
id.po: 301 translated messages, 8 fuzzy translations, 4 untranslated messages.
it.po: 81 translated messages, 232 untranslated messages.
ja.po: 81 translated messages, 232 untranslated messages.
ka.po: 56 translated messages, 2 fuzzy translations, 255 untranslated messages.
kk.po: 4 translated messages, 309 untranslated messages.
kn.po: 0 translated messages, 313 untranslated messages.
ko.po: 196 translated messages, 5 fuzzy translations, 112 untranslated messages.
lt.po: 22 translated messages, 291 untranslated messages.
lv.po: 80 translated messages, 233 untranslated messages.
ml.po: 0 translated messages, 313 untranslated messages.
mr.po: 0 translated messages, 313 untranslated messages.
ms.po: 0 translated messages, 313 untranslated messages.
nb.po: 0 translated messages, 313 untranslated messages.
nl.po: 80 translated messages, 233 untranslated messages.
nn.po: 0 translated messages, 313 untranslated messages.
oc.po: 81 translated messages, 232 untranslated messages.
or.po: 0 translated messages, 313 untranslated messages.
pa.po: 81 translated messages, 232 untranslated messages.
pl.po: 98 translated messages, 1 fuzzy translation, 214 untranslated messages.
pt_BR.po: 301 translated messages, 8 fuzzy translations, 4 untranslated messages.
pt.po: 81 translated messages, 232 untranslated messages.
ro.po: 0 translated messages, 313 untranslated messages.
ru.po: 81 translated messages, 232 untranslated messages.
si.po: 301 translated messages, 8 fuzzy translations, 4 untranslated messages.
sk.po: 81 translated messages, 232 untranslated messages.
sl.po: 81 translated messages, 232 untranslated messages.
sq.po: 0 translated messages, 313 untranslated messages.
sr@latin.po: 0 translated messages, 313 untranslated messages.
sr.po: 81 translated messages, 232 untranslated messages.
sv.po: 301 translated messages, 8 fuzzy translations, 4 untranslated messages.
ta.po: 0 translated messages, 313 untranslated messages.
te.po: 0 translated messages, 313 untranslated messages.
th.po: 0 translated messages, 313 untranslated messages.
tr.po: 81 translated messages, 232 untranslated messages.
uk.po: 301 translated messages, 8 fuzzy translations, 4 untranslated messages.
vi.po: 0 translated messages, 313 untranslated messages.
wa.po: 0 translated messages, 313 untranslated messages.
zh_CN.po: 86 translated messages, 1 fuzzy translation, 226 untranslated messages.
zh_HK.po: 0 translated messages, 313 untranslated messages.
zh_TW.po: 81 translated messages, 232 untranslated messages.
```

ueno reviewed (approved):
Thank you!

ueno commented:
I see the following error in the autotools build:
```console
si.1po:2: warning: header field 'Last-Translator' still has the initial default value
si.1po:654: format specifications in 'msgid' and 'msgstr' for argument 1 are not the same
/usr/bin/msgfmt: found 1 fatal error
si.1po: 301 translated messages.
make[3]: *** [Makefile:223: /__w/p11-kit/p11-kit/po/si.gmo] Error 1
```
@urbalazs could you take a look?

urbalazs commented:
@ueno Unfortunately Transifex has no checking tool if the placeholders are used different order in the translated string. I fixed the issue and added a dummy "Last-Traslator". Now both the warning and the fatal error should be gone and the build should work.

Since I changed a translation file, can you please push the changes to Transifex with `tx push -s`?

I suggest to remove the languages which has no translations at all. I guess you can do this in the web interface of Transifex, then you can remove the affected .po files and the corresponding entries in LINGUAS file.

By the way, Weblate is more powerful and you can use https://hosted.weblate.org/ free of charge for open source software.
